victor_description/launch/visualize_gripper.launch

```diff
@@ -0,0 +1,29 @@
+<!-- Note: This is not the normal path of execution to launch Victor -->
+<!-- This create a "robot" from the robotiq 3 finger gripper. -->
+<!-- This is useful for manually viewing the gripper in RViz when using something such as a grasp planner -->
+<!-- Place the gripper by publishing a transform from [baseframe] to [gripper_root]-->
+
+<launch>
+
+    <group ns="gripper_visual">
+        <arg name="model" default="$(find victor_description)/urdf/gripper_only.xacro"/>
+        <arg name="joint_state_publisher" default="true"/>
+        <arg name="gui" default="true"/>
+        <arg name="rviz" default="true"/>
```

Comment:
mild preference is for this arg to be false by default, but you could also just comment it out since rviz is also currently commented out